x/crosschain/keeper/msg_server_vote_outbound_tx.go

```diff
@@ -193,7 +193,10 @@ SaveFailedOutbound saves a failed outbound transaction.It does the following thi
 */
 
 func (k Keeper) SaveFailedOutbound(ctx sdk.Context, cctx *types.CrossChainTx, errMessage string, tssPubkey string) {
```

Comment:
We need to rename this, this is not the outbound that fails, where it would revert then but the observation of the outbound, meaning an issue at the protocol level

Reply:
Makes sense